tests/orbit/models/test_dlt.py

```diff
@@ -1,24 +1,28 @@
 import pytest
 import numpy as np
-import copy
+from copy import copy
 
-from orbit.models.dlt import BaseDLT, DLTFull, DLTAggregated, DLTMAP
+from orbit.models.dlt import DLTFull, DLTAggregated, DLTMAP
 from orbit.estimators.stan_estimator import StanEstimatorMCMC, StanEstimatorVI
 
 
-def test_base_dlt_init():
-    dlt = BaseDLT()
+@pytest.mark.parametrize("model_class", [DLTMAP, DLTFull, DLTAggregated])
+def test_base_ets_init(model_class):
+    lgt = model_class()
 
-    is_fitted = dlt.is_fitted()
+    is_fitted = lgt.is_fitted()
 
-    model_data_input = dlt._get_model_data_input()
-    model_param_names = dlt._get_model_param_names()
-    init_values = dlt._get_init_values()
+    model_data_input = lgt._get_model_data_input()
+    model_param_names = lgt._get_model_param_names()
+    init_values = lgt._get_init_values()
```

Comment:
should `lgt` be changed to `dlt` for consistency, since it is a unit test for dlt?